### PR TITLE
Port changes found in official trex image to Dockerfile

### PR DESCRIPTION
**Pull request by MehTheHedgehog**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:18.04
+MAINTAINER me@mehthehedgehog.be
+
+RUN apt-get update && \
+    apt-get install -y sudo gcc g++ python git zlib1g-dev pciutils vim kmod strace wget curl iproute2 tcpdump iputils-arping iputils-clockdiff iputils-ping iputils-tracepath inetutils-traceroute mtr && \
```
Comment (mkczuk) [inline Dockerfile]: split to shorter lines

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    rm -rf /var/lib/apt/lists/*
+
+ADD . /src/trex-core
```
Comment (mateumann) [inline Dockerfile]: COPY?

File: docker/etc/trex_cfg.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- port_limit    : 2
+  version       : 2
+  low_end       : true
+  interfaces    : ["veth0", "veth1"]   # list of the interfaces to bind run ./dpdk_nic_bind.py --status
```
Comment (mkczuk) [inline docker/etc/trex_cfg.yaml]: I'd remove this comment

File: docker/bin/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+ip link add veth0 type veth peer name veth1
+ip a add 1.1.1.1/255.255.255.0 dev veth0
+ip a add 2.2.2.2/255.255.255.0 dev veth1
```
Comment (mateumann) [inline docker/bin/entrypoint.sh]: would you consider changing those IPs to experimental? (e.g. 240.0.0.1/24 and 241.0.0.1/24)

File: docker/etc/trex_cfg.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  port_info     :  # set eh mac addr
+
+                 - ip         : 1.1.1.1
+                   default_gw : 2.2.2.2
+                 - ip         : 2.2.2.2
+                   default_gw : 1.1.1.1
```
Comment (mateumann) [inline docker/etc/trex_cfg.yaml]: ditto

File: docker/etc/trex_cfg.yaml (hunk 3 above)
Comment (MehTheHedgehog) [reply docker/etc/trex_cfg.yaml]: I've also removed comment below.

**Review by mateumann — APPROVED** [review]

**Review by mkczuk — APPROVED** [review]

**Comment by MehTheHedgehog** [comment]
[Add changes from trex-internal to our trex-core fork](https://trello.com/c/mMlgSMZ6/12-add-changes-from-trex-internal-to-our-trex-core-fork)

**Pull request merged**